psalm.xml.dist

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<psalm
+    totallyTyped="true"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://getpsalm.org/schema/config"
+    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+>
+    <projectFiles>
+        <directory name="bin"/>
+        <directory name="src"/>
+        <!-- <directory name="test"/> -->
```

Comment:
Tests should be covered by static analysis too: instead of removing coverage, let's use a baseline file (https://psalm.dev/docs/running_psalm/dealing_with_code_issues/)